src/evaluations/analyzer.py

```diff
@@ -107,16 +120,19 @@ def __init__(self, out_dir, evaluation_directory, evaluation_run_name,
         plots.
       evaluation_run_name: the run name of the evaluation.
       evaluation_name: the name of the evaluation config.
-      error_margin: a positive number setting the upper bound of the error. By
-        default, set to 0.05.
-      proportion_of_runs: a number between 0 and 1 that specifies the desired
-        proportion of runs within the error margin. By default, set to 0.95.
+      estimable_criteria_list: a list of tuples of error_margin and
+        proportion_of_runs. An error_margin is a positive number setting the
+        upper bound of the error, and the proportion_of_runs is a number
+        between 0 and 1 that specifies the desired proportion of runs within
+        the error margin.
       plot_params: a dictionary of the parameters of plot functions. If not
         given, will use PLOT_PARAMS. Also see PLOT_PARAMS for how it is defined.
     """
-    self.error_margin = error_margin
-    self.proportion_of_runs = proportion_of_runs
-    self.plot_params = plot_params or PLOT_PARAMS
+    self.estimable_criteria_list = estimable_criteria_list
+    if plot_params is None:
+      self.plot_params = PLOT_PARAMS
+    else:
+      self.plot_params = plot_params
 
     # Get all the raw results.
     self.evaluation_file_dirs = evaluator.load_directory_tree(
```

Comment:
Seems `evaluation_directory` is passed here, but  `os.path.join(out_dir, run_name)` in `load_directory_tree` would throw exception if it's null

Reply:
sure good idea

Reply:
true but since there's not a default value in the constructor I don't think it's a worry that evaluation_directory would ever be None